editor/src/clj/editor/code/view.clj

```diff
@@ -3701,9 +3782,18 @@
                                             cursor-range))))
 
 (defn register-view-types [workspace]
-  (workspace/register-view-type workspace
-                                :id :code
-                                :label "Code"
-                                :make-view-fn (fn [graph parent resource-node opts] (make-view! graph parent resource-node opts))
-                                :focus-fn (fn [view-node opts] (focus-view! view-node opts))
-                                :text-selection-fn non-empty-single-selection-text))
+  (concat
+    (workspace/register-view-type workspace
+      :id :code
+      :label "Code"
+      :make-view-fn make-view!
+      :focus-fn focus-view!
+      :text-selection-fn non-empty-single-selection-text)
+    (workspace/register-view-type workspace
+      :id :text
+      :label "Text"
+      :make-view-fn make-view!
+      :focus-fn focus-view!
+      :text-selection-fn non-empty-single-selection-text)))
```

Comment:
We now register both the `:code` and the `:text` view types to use the `CodeEditorView`.